main.go

```diff
@@ -1054,7 +1046,7 @@ func tabPrintSortedMap(headers []string, m map[string]string) {
 func getGoPath() (string, error) {
 	gp := os.Getenv("GOPATH")
 	if gp == "" {
-		return "", fmt.Errorf("GOPATH not set")
+		return homedir.Expand("~/go")
```

Comment:
Hence you probably set GOPATH to something else :) This change just mirrors the default assumed by the go tool.